### PR TITLE
AB#4255 Add more specific `Policy` classes to DeviceConfig and CompliancePolicy entities

### PR DESCRIPTION
**Pull request by mknoedel**

### CODE REVIEW

File: src/steps/intune/clients/deviceManagementIntuneClient.ts [hunk 1]
```diff
@@ -135,6 +135,11 @@ export class DeviceManagementIntuneClient extends GraphClient {
     });
   }
 
+  //***** CONDITIONAL ACCESS POLICIES *********/
```
Comment (mknoedel) [inline src/steps/intune/clients/deviceManagementIntuneClient.ts:138]: We will likely need to ingest these at some point in the future.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ and this project adheres to
 
 ### Added
 
+- A class of `CompliancePolicy` to `intune_compliance_policy` and
+  `intune_device_configuration` entities
+- `categpory` property to `intune_compliance_policy` and
```
Comment (aiwilliams) [inline CHANGELOG.md:15]:
```suggestion
- `category` property to `intune_compliance_policy` and
```

File: src/steps/intune/constants.ts [hunk 3]
```diff
@@ -49,12 +49,12 @@ export const entities = {
   DEVICE_CONFIGURATION: {
     resourceName: 'Device Configuration',
     _type: 'intune_device_configuration',
-    _class: 'Configuration',
+    _class: ['Configuration', 'ControlPolicy'] as string[],
```
Comment (aiwilliams) [inline src/steps/intune/constants.ts:52]: I would recommend that we always use a `string[]` even in cases where there is only a single value. That would avoid the need to cast here.

File: src/steps/intune/steps/compliancePoliciesAndFindings/converters.ts [hunk 4]
```diff
@@ -34,6 +34,7 @@ export function createCompliancePolicyEntity(
         _class: entities.COMPLIANCE_POLICY._class,
         _type: entities.COMPLIANCE_POLICY._type,
         function: 'endpoint-compliance', // needed to match up with existing integrations data structures.
+        category: 'compliance', // needed for "ControlPolicy"
```
Comment (aiwilliams) [inline src/steps/intune/steps/compliancePoliciesAndFindings/converters.ts:37]: Nice.

**Review by aiwilliams — APPROVED** [review]
There's a typo, but that can be fixed when versioning.

**Pull request merged**